### ISSUE TITLE
[Feature] Activate the NextChat's chat window or mouse cursor 

### ISSUE THREAD

**Issue by smjure** [ChatGPTNextWeb/ChatGPT-Next-Web#3960]
I use NextChat Desktop app on Linux and I basically never close it. It happens many times, that when I bring up the NextChat app window, the chat is not activated, which I have to do then manually by help of my mouse.

Is it possible that when you bring the NextChat window to front on your Desktop, the chat is automatically activated, so one can just start typing. Alternatively, make a keybinding to make an inactive chat window active. Thanks and thanks for the great app :bow: 🙏

**Comment by H0llyW00dzZ** [comment]
> I use NextChat Desktop app on Linux and I basically never close it. It happens many times, that when I bring up the NextChat app window, the chat is not activated, which I have to do then manually by help of my mouse.
> 
> Is it possible that when you bring the NextChat window to front on your Desktop, the chat is automatically activated, so one can just start typing. Alternatively, make a keybinding to make an inactive chat window active. Thanks and thanks for the great app 🙇 🙏

you mean like this ?

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/8da47ba0-9c94-4cb0-8876-325935650848)

**Comment by smjure** [comment]
Yes, if I see it correctly, the mouse cursor is not active in your chat window. The idea is to make it automatically active when bringing the NextChat app to the front, so we can just start typing. Sorry, I tried to make print screen but the active cursor is for some reason not shown in my case.

**Comment by H0llyW00dzZ** [comment]
Link Download: [`here`](https://efss.qloud.my/index.php/s/ZNiYY2T7JK2oH5x) (currently only windows)

> [!NOTE]  
> Don't forget change the url end point, since it using [`my own proxy`](https://hackerchat.btz.sh/api/proxy)

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/ba3336fc-8200-429c-8715-3d69997dffe4)

**Comment by smjure** [comment]
Thanks for the efforts, but am only using Linux. Hopefully, this feature comes in the future, either as automatic or as  a possibility to set it as a keybinding.

**Comment by H0llyW00dzZ** [comment]
> Thanks for the efforts, but am only using Linux. Hopefully, this feature comes in the future, either as automatic or as a possibility to set it as a keybinding.

are you on ubuntu ? if on ubuntu might I can build it through CI/CD

**Comment by smjure** [comment]
Linux Mint (Edge), basically yes Ubuntu 🙇‍♀️🙏

**Comment by H0llyW00dzZ** [comment]
> Linux Mint (Edge), basically yes Ubuntu 🙇‍♀️🙏

Unfortunately I can't build it, the rust compiler is so slow
